etl-repair.md

```diff
@@ -69,6 +69,19 @@ $ kubectl logs deploy/kubecost-cost-analyzer | grep CloudCost
 ```
 {% endcode %}
 
+By default, all users v1.106+ no longer have [CloudUsage ETL](https://github.com/kubecost/cost-analyzer-helm-chart/blob/63dea06ae1ea771a8622e786b22f6f6625a49349/cost-analyzer/templates/cost-analyzer-deployment-template.yaml#L788-L794) enabled. If you are using CloudUsage ETL, use the commands below:
```

Comment:
I really dislike mentioning specific line numbers in any code in the documentation because it almost immediately gets outdated. I instead like to mention which value or, just generally, which functionality may need to be enabled and point to the file and let users find the exact location.

Reply:
@chipzoller Removed the link! Replaced with helm config value.